script/precompile-assets.js

```diff
@@ -62,6 +75,53 @@ function precompileCss() {
   })
 }
 
+async function precompileImages() {
+  const websiteImages = await globby(`${PATHS.imagesDir}/**/*`, {
```

Comment:
```suggestion
  const websiteImages = await globby(path.join(PATHS.imagesDir, '**', '*'), {
```

We can support windows here 😄

Reply:
WSL has spoiled me 😅